suites/pacific/rados/tier-3_rados_ssd-tests.yaml

```diff
@@ -154,22 +191,22 @@ tests:
   - test:
       name: OSD deployment on single SSD
       desc: Add OSD services on single SSD.
-      module: test_daemon.py
       polarion-id: CEPH-9268
+      module: test_add_osd_ssd.py
       config:
-        steps:
+        add:
+          - config:
```

Comment:
Let's keep only Nvme drives for addition. Reason being there is a chance that `/dev/sdm` or `/sdl` can be used for OS post reimage, and the test can fail.

Reply:
Modified the logic in such a way that I am providing node names as the inputs. The script checks the which ever the SSD/NVMES are available  on that device paths the OSD will get configured.

 _name: OSD deployment on single SSD
      desc: Add OSD services on single SSD.
      polarion-id: CEPH-9268
      module: test_add_osd_ssd.py
      config:
        add:
          - config:
              node_name: node14
          - config:
              node_name: node15_

Reply:
Sounds good